### PR TITLE
Ch modal

### PR DESCRIPTION
**Pull request by jothamardel**
## Description

- Modal reused for feedback, update user profile and victims info 

- Added error handling 

- Redux setup for update profile

### CODE REVIEW

**Review by nonsoo24 — CHANGES_REQUESTED** [review]
Resolve conflict

**Review by Ose-Me — APPROVED** [review]

**Review by jason-ibisi — APPROVED** [review]

**Review by nonsoo24 — APPROVED** [review]

**Comment by Ose-Me** [comment]
image

**Comment by Ose-Me** [comment]
that's how the homepage looks on my screen
and also fix these issues https://app.codacy.com/gh/BuildForSDG/Team-075-ERS/pullRequest?prid=5595115#newIssuesView

File: src/components/nav-bar/Navbar.jsx [hunk 1]
```diff
@@ -2,9 +2,10 @@ import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
 import { connect } from 'react-redux';
 import { helpSent } from '../../redux/sendHelp/sendHelp.actions';
-import { logoutUser } from '../../redux/user/user.actions';
-import CustomButton from '../custom-button/CustomButton';
-import './navbar.css';
```
Comment (Ose-Me) [inline src/components/nav-bar/Navbar.jsx:7]: you removed the navbar import statement here

**Comment by jothamardel** [comment]
This branch has bugs but #54 has solved it. Should I go ahead and close this PR? @Ose-Me, @jason-ibisi, @nonsoo24  and @oluwaseunsamuel .

**Comment by jason-ibisi** [comment]
> This branch has bugs but #54 has solved it. Should I go ahead and close this PR? @Ose-Me, @jason-ibisi, @nonsoo24 and @oluwaseunsamuel .

If #54 has resolved the issues on this branch please close this issue and delete the branch

**Review by nonsoo24 — APPROVED** [review]

**Pull request closed without merging**